network/shuffle/src/test/java/org/apache/spark/network/shuffle/RetryingBlockFetcherSuite.java

```diff
@@ -54,7 +54,7 @@
   @Before
   public void beforeEach() {
     System.setProperty("spark.shuffle.io.maxRetries", "2");
-    System.setProperty("spark.shuffle.io.retryWait", "0");
+    System.setProperty("spark.shuffle.io.retryWait", "0s");
```

Comment:
No need to change this, but as an observation, "0" ought to be fine everywhere without units since it's unambiguous.